bpf/aya-bpf/src/args.rs

```diff
@@ -1,3 +1,5 @@
+use aya_bpf_bindings::bindings::__u64;
```

Comment:
I would rather use `::aya_bpf_cty::c_ulonglong` directly instead of using this type with underscore.

Reply:
good idea!